api/src/main/java/org/apache/iceberg/types/TypeUtil.java

```diff
@@ -712,6 +712,10 @@ public T map(Types.MapType map, Supplier<T> keyResult, Supplier<T> valueResult)
       return null;
     }
 
+    public T variant() {
+      return null;
```

Comment:
Looks like this is still null. The one above is updated though.

Reply:
I missed that.